io/disk/Avago_storage_adapter/avago3008.py.data/avago3008.yaml

```diff
@@ -1,8 +1,9 @@
 cenario:
     contorller: 0
-    disk: 1:4 1:5 1:6
-    spare: "1:7"
+    disk: 2:1 2:4 2:5 2:6
```

Comment:
Can we leave `disk` and `spare` empty, so that users who run by mistake do not suffer from loss of disk data.